packages/monaco-graphql/test/monaco-editor.test.ts

```diff
@@ -13,7 +13,7 @@ describe('monaco-editor', () => {
     // expect(lines[0]).toBe('$ vite build');
     // expect(lines[1]).toMatch(' building for production...');
     // expect(lines[2]).toBe('transforming...');
-    expect(lines[3]).toMatch('✓ 970 modules transformed.');
+    expect(lines[3]).toMatch('✓ 839 modules transformed.');
```

Comment:
reduces the total modules by replacing `edcore.main.js`, even though we add the typescript mode to the  vite bundle!